### DISCUSSION TITLE
Tutorial - Security First Step tutorial doesn't work as mentioned

### DISCUSSION THREAD

**Discussion by 2kiranns** [fastapi/fastapi#9583] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# https://fastapi.tiangolo.com/tutorial/security/first-steps/

from typing import Annotated

from fastapi import Depends, FastAPI
from fastapi.security import OAuth2PasswordBearer

app = FastAPI()

oauth2_scheme = OAuth2PasswordBearer(tokenUrl="token")


@app.get("/items/")
async def read_items(token: Annotated[str, Depends(oauth2_scheme)]):
    return {"token": token}
```

### Description

The tutorial code mentioned in https://fastapi.tiangolo.com/tutorial/security/first-steps/ does not work as expected. When I navigate to http://localhost:8000/docs it does not show up the "Authorize" button.

### Operating System

Windows

### FastAPI Version

0.91.0

### Python Version

3.11.2

### Additional Context

Authorization form is not appearing as mentioned in the website. It is expecting token as a URL parameter. Something like http://localhost:8000/items/?token=asdasd

**Comment by jgould22** [comment, marked as answer]
You appear to be using an older version of FastAPI.

Support for annotated was added in 0.95.0 so I would first start with updating your fast api version and see if that fixes it.

**Reply by 2kiranns** [reply]
I upgraded to 0.95.1 and the issue is address. Thank you so much. If we can update the tutorial with minimum version required it helps.
